androgee/androgee.py

```diff
@@ -6,23 +6,26 @@
 
 
 logging.basicConfig(level=logging.WARNING)
-bot = commands.Bot(command_prefix=os.environ['DISCORD_PREFIX'])
+bot = commands.Bot(command_prefix=os.environ["DISCORD_PREFIX"])
+
 
 @bot.event
 async def on_ready():
-    print('We is logged in as {0.user}'.format(bot))
+    print("We is logged in as {0.user}".format(bot))
```

Comment:
Oh do you mind adding a Flake8 rule or even a Git precommit hook for checking this? You can create a new PR for it or add it to this, your choice.

Reply:
Context - a lint rule for ' to "

Reply:
I would actually recommend using a formatter like black to do this I'll open another pr if you would prefer flake8 rule

Reply:
Submit a PR for Black so I can see

Reply:
I have submitted a pr for black